src/overlays/actors/ovl_Obj_Roomtimer/z_obj_roomtimer.c

```diff
@@ -45,14 +45,14 @@ void ObjRoomtimer_Init(Actor* thisx, PlayState* play) {
 void ObjRoomtimer_Destroy(Actor* thisx, PlayState* play) {
     ObjRoomtimer* this = THIS;
 
-    if (this->actor.params != 0x1FF && gSaveContext.unk_3DD0[4] > 0) {
-        gSaveContext.unk_3DD0[4] = 5;
+    if ((this->actor.params != 0x1FF) && (gSaveContext.timerStates[TIMER_ID_MINIGAME_2] >= TIMER_STATE_START)) {
```

Comment:
Not necessarily, room-timer still functions as it does in OoT afaik. It's just that for the two timer Id's `TIMER_ID_MINIGAME` and `TIMER_ID_MINIGAME_2`, they both seemed to be used entirely for minigames, both are used many times, and I wasn't able to yet decern the difference between the two. This seems to be the one exception that the Id commonly used for many minigames gets used for "room-timers". So it would take another round of docs, plus 100% decomp, to work out the full scope of these two timers